### ISSUE TITLE
Display HTML redirects on the index page

### ISSUE THREAD

**Issue by FafnerKeyZee** [CIRCL/lookyloo#65]
More a request than a real bug, but this can help in some cases.
Some websites do redirection from header if the page is called by command-line tools (curl/wget) or from html redirect when the user uses a browser.

ie: t.co
via curl:
```
curl https://t.co/REMOVED?amp=1 -I

HTTP/2 301
cache-control: private,max-age=300
date: Thu, 13 Feb 2020 10:04:15 GMT
expires: Thu, 13 Feb 2020 10:09:15 GMT
location: https://REMOVED
server: tsa_f
set-cookie: muc=68b15902-7ff7-4765-84d8-d783ade72374; Max-Age=63072000; Expires=Sat, 12 Feb 2022 10:04:15 GMT; Domain=t.co
strict-transport-security: max-age=0
vary: Origin
x-connection-hash: 504674ad9a6cee999a3d2f4e2ed9c40d
x-response-time: 115
```
via browser (splash output)
```
<head><noscript>
<META http-equiv="refresh" content="0;URL=https://pastebin.com/<dsfs>">
</noscript>
<title>https://pastebin.com/<dsfs></title></head>
<script>window.opener = null; location.replace("https:\/\/pastebin.com\/<dsfs>")</script>
```

The goal of the issue is to display the redirect in the main page, and in the tree page too.

**Comment by Rafiot** [comment]
Sooo just a note: it is pretty easy to find the meta refresh stuff, the pure javascript calls are way more complicated to catch in a generic manner, as it can be obfuscated in all kind of hilarious ways.

**Comment by FafnerKeyZee** [comment]
At least we can try to check the domain ? dunno what you receive in har return

**Comment by Rafiot** [comment]
We have all the URLs loaded in the HAR, but making the difference between a page component loaded in JS, and a redirect is non trivial. I'll try to come with something.

**Comment by Rafiot** [comment]
Note: It's impossible to inject javascript and have it reliably working: https://github.com/scrapinghub/splash/issues/313

TL;DR: if CSP is enabled on the target website, splash barf an exception.

**Comment by Rafiot** [comment]
Note, take 2: The following call, before returning will five the last href, after redirect.
`local last_location = splash:evaljs("location.href;")`

**Comment by Rafiot** [comment]
Annnd one more note: https://splash.readthedocs.io/en/stable/scripting-ref.html#splash-url

Works, and CSP doesn't complain \o/

**Comment by Rafiot** [comment]
That should fix it: https://github.com/CIRCL/lookyloo/commit/971877954e919423cb4f99a77c8f2c6ad5808b48
